packages/css-jss/src/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import {create as createJss} from 'jss'
+import type {StyleSheet} from 'jss'
+import preset from 'jss-preset-default'
+import createCss from './createCss'
+import type {Css, Style} from './types'
+
+const jss = createJss(preset())
+const defaultSheet = jss.createStyleSheet().attach()
+
+export type {Css, Style}
+
+export const create = (sheet?: StyleSheet): Css => {
+  if (sheet) defaultSheet.detach()
```

Comment:
Why would we detach in such a case? The user still could have used the default sheet

Reply:
Yeah, not sure what to do here, the problem I see is that if user creates custom jss/sheet, this sheet will already be in the DOM with no rules.

So maybe I need to additionally check if there are rules

Reply:
Would this be a big benefit? I don’t think it will make a big difference in performance

Reply:
Just the fact that there would be an empty style tag in the head, though evtl it doesn't matter for client side and won't happen on the server so fuck it, I am removing the line.